bamboo/unit_tests/test_unit_layer_gather.py

```diff
@@ -18,10 +18,10 @@
 # the functions below to ingest data.
 
 # Data
-num_rows = 5
-num_columns = 20
+num_rows = 10
```

Comment:
It's worrying that this test didn't catch some correctness bugs. Maybe the tensor dimensions were too nice.
```suggestion
num_rows = 11
```